packages/angular-playground/CHANGELOG.md

```diff
@@ -1,3 +1,10 @@
+# 5.8.2 (2019-9-24)
```

Comment:
This is the wrong version info.  I don't think you updated your master branch with origin before creating your branch.